modules/geo-layers/src/tile-layer/tile-layer.ts

```diff
@@ -165,7 +165,7 @@ export default class TileLayer<DataT = any, ExtraPropsT extends {} = {}> extends
   }
 
   get isLoaded(): boolean {
-    return this.state?.tileset?.selectedTiles.every(
+    return this.state?.tileset?.selectedTiles?.every(
```

Comment:
Question: would it be worth it to use something like `...tileset?.isLoaded`? At a quick glance the potential tradeoffs and subtle differences aren't fully clear to me yet.

https://github.com/visgl/deck.gl/blob/master/modules/geo-layers/src/tileset-2d/tileset-2d.ts#L183-L185